crates/wdk-sys/Cargo.toml

```diff
@@ -74,4 +74,4 @@ redundant_explicit_links = "warn"
 unescaped_backticks = "warn"
 
 [package.metadata.cargo-machete]
-ignored = ["lazy_static"] # lazy_static is used in code generated by build.rs
```

Comment:
keep the comment

Reply:
this entire line is gone -- there's no reference to lazy_static. why do we need to keep the comment?

Reply:
there is still a machete entry for once_cell. I want a comment documenting why machete needs to ignore once_cell (presumably because its only used in generated code that machete doesn't scan , resulting in a false positive).

Reply:
without the comment, machete fails, right?